boards/arm/udoo_neo_full_m4/udoo_neo_full_m4_defconfig

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_ARM=y
+CONFIG_SOC_MCIMX6X_M4=y
+CONFIG_SOC_SERIES_IMX_6X_M4=y
+CONFIG_BOARD_UDOO_NEO_FULL_M4=y
+CONFIG_CONSOLE=y
+CONFIG_CONSOLE_HAS_DRIVER=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
+CONFIG_SERIAL_HAS_DRIVER=y
+CONFIG_CORTEX_M_SYSTICK=y
+CONFIG_GPIO=n
```

Comment:
In the board dts you set the gpio ports 4, 5 and 6 to status "ok" but in this config the GPIO is set to "n".

Reply:
Changed.